certified-connectors/Plumsail Forms/apiDefinition.swagger.json

```diff
@@ -1,14 +1,14 @@
 {
   "swagger": "2.0",
   "info": {
-    "version": "v1",
     "title": "Plumsail Forms",
     "description": "This API allows you to process Plumsail Forms submissions and their attachments.",
     "contact": {
       "name": "Plumsail Team",
       "url": "https://plumsail.com",
       "email": "support@plumsail.com"
-    }
+    },
+    "version": "v1"
```

Comment:
@jusper-dk the latest version in ISV Studio for Plumsail Forms is version 4.0. if this is a new version, please change to "5.0".

Reply:
Done.